examples/chaincode/go/rbac/README.md

```diff
@@ -0,0 +1,82 @@
+# Hyperledger Fabric - Role-based Access Control
+
+## Overview
+
+The *rbac* chaincode (*rbac.go*) is a very simple chaincode designed to show how to exercise *robe-based access control* at the chaincode level by leveraging the techniques described in this document: [https://github.com/hyperledger/fabric/blob/master/docs/tech/application-ACL.md](https://github.com/hyperledger/fabric/blob/master/docs/tech/application-ACL.md)
+
+The chaincode exposes the following functions:
+
+1. *init(user)*: Initialize the chaincode assigning to *user* the role of *administrator*;
+2. *addRole(user, role)*: Assigns the role *role* to *user*. 
+Notice that, this function can be invoked only by an administrator;
+3. *write(data)*: Set the current chaincode's state to *data*.
+Notice that this function can be invoked only by users who have the role *writer*;
+4. *read()*: Returns the current chaincode's state.
+Notice that this function can be invoked only by users who have the role *reader*;
+
+In the following subsections, we will describe in more detail each function.
+
+## *init(user)*
+
+This function initialize the chaincode by assigning to *user* the role of administrator. The function is invoked automatically at deploy time.
+
+When generating the deploy transaction, the chaincode deployer must specify the administrator of the chaincode by setting the transaction metadata to 
+the DER (Distinguished Encoding Rules) certificate encoding of one of the administrator ECert/TCert. 
+
+For simplicity, there is only one administrator.
+
+A possible work-flow could be the following:
+
+1. Alice is the deployer of the chaincode;
+2. Alice wants to assign the administrator role to Bob;
+3. Alice obtains, via an out-of-band channel, a TCert of Bob, let us call this certificate *BobCert*;
+4. Alice constructs a deploy transaction, as described in *application-ACL.md*,  setting the transaction metadata to *DER(BobCert)*.
```

Comment:
Sorry the suggestion wasn't clear. I meant to suggest: "as described in [Application ACLs](https://github.com/hyperledger/fabric/blob/master/docs/tech/application-ACL.md)"
 instead of "as described in **\*application-ACL.md***" [in line 33]

That is:  \[Application ACLs\] \(https://github.com/hyperledger/fabric/blob/master/docs/tech/application-ACL.md)